modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/fragment/renderer/FDSViewFragmentRenderer.java

```diff
@@ -260,6 +260,20 @@ private String _buildFragmentHTML(
 			fragmentEntryLink.getCompanyId(), fdsEntryObjectEntryERC,
 			fdsEntryObjectDefinition);
 
+		Collection<ObjectEntry> fdsCardObjectEntries = null;
+
+		Collection<ObjectEntry> fdsListObjectEntries = null;
+
+		if (FeatureFlagManagerUtil.isEnabled("LPD-10735")) {
+			fdsCardObjectEntries = _getRelatedObjectEntries(
+				fdsViewObjectDefinition, fdsViewObjectEntry,
+				"fdsViewFDSCardsSectionRelationship");
+
+			fdsListObjectEntries = _getRelatedObjectEntries(
+				fdsViewObjectDefinition, fdsViewObjectEntry,
+				"fdsViewFDSListSectionRelationship");
+		}
+
 		Set<ObjectEntry> fdsFieldObjectEntries = _getFDSFieldObjectEntries(
```

Comment:
Maybe it is time to rename this `fdsFieldObjectEntries`  to `fdsTableObjectEntries`

Reply:
Don't tell anybody, but I'm planning to sneak this in as a part of the story that removes `FDSViews` :shushing_face:   I'm thinking `FDSTableColumn`, or even `DSMTableColumn`.